easypy/humanize.py

```diff
@@ -16,6 +16,7 @@
 from easypy.bunch import Bunch, bunchify
 from easypy.colors import colorize
 from easypy.collections import ilistify
+from easypy.humanize import clamp
```

Comment:
1. This already is `easypy.humanize`.
2. Didn't you just delete `clamp` in this very PR? From **both** `easypy.humanize` and `easypy.random`?